### PR TITLE
several update for v2v tests

### PR DESCRIPTION
**Pull request by will-Do**
Please check each commit for detail.

### CODE REVIEW

File: virttest/utils_test/libvirt.py [hunk 1]
```diff
@@ -73,55 +73,60 @@ def create_vnet_xml(self):
...
+        address = self.kwargs.get('address')
+        if not address:
+            raise exceptions.TestError('Create vnet need address be set')
         net_xml = NetworkXML()
         net_xml.name = self.name
-        ip = IPXML(address=self.address)
+        ip = IPXML(address=address)
+        dhcp_start = self.kwargs.get('dhcp_start')
+        dhcp_end = self.kwargs.get('dhcp_end')
+        if all('dhcp_start', 'dhcp_end'):
```
Comment (waynesun09) [inline virttest/utils_test/libvirt.py]: all take one argument on python 2.x

Comment (will-Do) [inline virttest/utils_test/libvirt.py]: Yes, it's a wrong usage, just fix it.

**Comment by waynesun09** [comment]
ACK

**Comment by rbian** [comment]
ACK

**Comment by will-Do** [comment]
Thx all

**Pull request merged**